themes/10up-theme/includes/blocks.php

```diff
@@ -90,6 +92,45 @@ function blocks_editor_styles() {
 
 }
 
+/**
+ * Allowed blocks in editor
+ * See reference of available blocks here: https://wordpress.org/support/article/blocks/
+ *
+ * @param array $allowed_block_types core array
+ * @return array  $allowed_blocks modified array under right conditions.
+ */
+function allowed_block_types( $allowed_block_types ) {
+
+	$allowed_blocks = [
+		'core/paragraph',
```

Comment:
@joesnellpdx where is this list from? For example, I see that core/embed is missing and YouTube embeds are one of the most commonly used blocks. I def think this allowed block filter is useful but should make sure we don't blacklist commonly used blocks out of the box.

Also, include a comment or even add the 10up-plugin (mu-plugins/10up-plugin) commented out to this list as an example of including custom blocks?

Reply:
@smy315  List came from UX team and some others. You are right, embed should be in there - added that back. 

Great advice to also add better guidance for block library blocks and custom blocks - this has been updated.